instrumentation/netty/netty-4.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/netty/v4_0/BootstrapInstrumentation.java

```diff
@@ -32,16 +31,16 @@
   @Override
   public void transform(TypeTransformer transformer) {
     transformer.applyAdviceToMethod(
-        named("doConnect")
-            .and(takesArgument(0, SocketAddress.class))
-            .and(returns(named("io.netty.channel.ChannelFuture"))),
+        named("doConnect0")
```

Comment:
Just out of curiosity - why this change?

Reply:
The `doConnect` actually traces more than just connection - it starts with tracing channel registration (which happens before) and ends after handler `channelActive()` has returned (which is way after SSL handshake is finished). The `doConnect0` method seems to be doing just the connection, which is more accurate.